Library/Homebrew/service.rb

```diff
@@ -391,6 +408,7 @@ def to_plist
       base[:LegacyTimers] = @macos_legacy_timers if @macos_legacy_timers == true
       base[:TimeOut] = @restart_delay if @restart_delay.present?
       base[:ProcessType] = @process_type.to_s.capitalize if @process_type.present?
+      base[:Nice] = @nice if @nice.present?
```

Comment:
```suggestion
      base[:Nice] = @nice if @nice
```
`present?` is generally unnecessary redirection, unless whitespace is concerned. (It's either `nil` or `Integer` if we get here, correct?)